media-sound/qmmp/qmmp-2.2.3.ebuild

```
@@ -0,0 +1,173 @@
+
+DESCRIPTION="Qt6-based audio player with winamp/xmms skins support"
+HOMEPAGE="https://qmmp.ylsoftware.com"
+if [[ ${PV} != *9999* ]]; then
+	SRC_URI="
+		https://qmmp.ylsoftware.com/files/${PN}/$(ver_cut 1-2)/${P}.tar.bz2
+		https://downloads.sourceforge.net/${PN}-dev/files/${P}.tar.bz2
+	"
+	KEYWORDS="~amd64 ~x86"
+else
+	inherit subversion
+	local BRANCH_VER="2.2" # NOTE: bump this to $(ver_cut 1-2) for live ebuild
+	ESVN_REPO_URI="svn://svn.code.sf.net/p/qmmp-dev/code/branches/${PN}-${BRANCH_VER}"
+fi
+
+LICENSE="CC-BY-SA-4.0 GPL-2+" # default skin & source code
+SLOT="0"
+# KEYWORDS further up
+IUSE="X aac +alsa archive bs2b cdda cddb curl +dbus
+doc enca ffmpeg flac game gnome jack ladspa libxmp +mad midi mms mpg123
+mplayer musepack opus pipewire projectm pulseaudio qsui qtmedia
+shout sid sndfile soxr udisks +vorbis wavpack"
+
+REQUIRED_USE="
+	cddb? ( cdda )
+	gnome? ( dbus )
+	shout? ( soxr vorbis )
+	udisks? ( dbus )
+"
+# qtbase[sql] to help autounmask of sqlite
+RDEPEND="
+	dev-qt/qtbase:6[X?,dbus?,gui,network,sql,sqlite,widgets]
+	media-libs/taglib:=
+	X? ( x11-libs/libX11 )
+	aac? ( media-libs/faad2 )
+	alsa? ( media-libs/alsa-lib )
+	archive? ( app-arch/libarchive )
+	bs2b? ( media-libs/libbs2b )
+	cdda? (
+		dev-libs/libcdio:=
+		dev-libs/libcdio-paranoia:=
+	)
+	cddb? ( media-libs/libcddb )
+	curl? ( net-misc/curl )
+	enca? ( app-i18n/enca )
+	ffmpeg? ( media-video/ffmpeg:= )
+	flac? ( media-libs/flac:= )
+	game? ( media-libs/game-music-emu )
+	jack? (	virtual/jack )
+	ladspa? ( media-plugins/cmt-plugins )
+	libxmp? ( media-libs/libxmp )
+	mad? ( media-libs/libmad )
+	midi? ( media-sound/wildmidi )
+	mms? ( media-libs/libmms )
+	mpg123? ( media-sound/mpg123 )
+	mplayer? ( media-video/mplayer )
+	musepack? ( >=media-sound/musepack-tools-444 )
+	opus? ( media-libs/opusfile )
+	pipewire? ( media-video/pipewire:= )
+	projectm? (
+		dev-qt/qtbase:6[-gles2-only,opengl]
+		media-libs/libprojectm:=
+	)
+	pulseaudio? ( media-libs/libpulse )
+	qtmedia? ( dev-qt/qtmultimedia:6 )
+	scrobbler? ( net-misc/curl )
+	shout? ( media-libs/libshout )
+	sid? ( >=media-libs/libsidplayfp-1.1.0:= )
+	sndfile? ( media-libs/libsndfile )
+	soxr? ( media-libs/soxr )
+	udisks? ( sys-fs/udisks:2 )
+	vorbis? (
+		media-libs/libogg
+		media-libs/libvorbis
+	)
+	wavpack? ( media-sound/wavpack )
+"
+DEPEND="
+	${RDEPEND}
+	X? ( x11-base/xorg-proto )
+"
+BDEPEND="
+	dev-qt/qttools:6[linguist]
+	doc? ( app-text/doxygen )
+"
+
+DOCS=( AUTHORS ChangeLog README )
+
+PATCHES=( "${FILESDIR}"/${P}_build-with-projectm4.patch )
+
+src_configure() {
+	local mycmakeargs=(
+		# our defaults
+		-DUSE_CONVERTER=ON # because taglib
+		-DUSE_RGSCAN=ON # because taglib
+		-DUSE_LYRICS=ON # because qtbase[network]
+		-DUSE_SCROBBLER=ON # because qtbase[network]
+		-DUSE_LIBRARY=ON # because qtbase[sqlite]
+
+		# depless options
+		-DUSE_ANALYZER=ON
+		-DUSE_COVER=ON
+		-DUSE_CROSSFADE=ON
+		-DUSE_CUE=ON
+		-DUSE_OSS=ON
+		-DUSE_OSS4=ON
+		-DUSE_STATICON=ON
+		-DUSE_STEREO=ON
```

```suggestion
		# depless options
		-DUSE_OSS=ON
```
keep only non default.
OSS4 can't be enabled because it requires a lib not in tree